.pre-commit-config.yaml

```diff
@@ -43,11 +44,6 @@ repos:
   - id: black
     args: [--config=pyproject.toml, --verbose]
 
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v1.4.1
-  hooks:
-  - id: mypy
```

Comment:
My guess is this previously worked by accident. Here's what you get now:

```
$ mypy --explicit-package-bases bids/ext/reports
bids/ext/reports/templates/demos/python/demo.py:5: error: Module "bids" has no attribute "BIDSLayout"  [attr-defined]
bids/ext/reports/templates/demos/python/demo.py:10: error: Function is missing a return type annotation  [no-untyped-def]
bids/ext/reports/templates/demos/python/demo.py:14: error: Function is missing a type annotation  [no-untyped-def]
bids/ext/reports/templates/demos/python/demo.py:21: error: Function is missing a type annotation  [no-untyped-def]
bids/ext/reports/templates/demos/python/demo.py:39: error: Function is missing a return type annotation  [no-untyped-def]
bids/ext/reports/templates/demos/python/demo.py:39: note: Use "-> None" if function does not return a value
bids/ext/reports/utils.py:11: error: Cannot find implementation or library stub for module named "bids.layout"  [import]
bids/ext/reports/parameters.py:9: error: Cannot find implementation or library stub for module named "num2words"  [import]
bids/ext/reports/parameters.py:15: error: Cannot find implementation or library stub for module named "bids.layout"  [import]
bids/ext/reports/parameters.py:56: error: Argument 1 to "append" of "list" has incompatible type "None"; expected "int"  [arg-type]
bids/ext/reports/parameters.py:71: error: Argument 1 to "get_nb_vols" has incompatible type "List[Nifti1Image]"; expected "List[Optional[Nifti1Image]]"  [arg-type]
bids/ext/reports/parameters.py:71: note: "List" is invariant -- see https://mypy.readthedocs.io/en/stable/common_issues.html#variance
bids/ext/reports/parameters.py:71: note: Consider using "Sequence" instead, which is covariant
bids/ext/reports/parameters.py:80: error: Argument 1 to "get_nb_vols" has incompatible type "List[Nifti1Image]"; expected "List[Optional[Nifti1Image]]"  [arg-type]
bids/ext/reports/parameters.py:80: note: "List" is invariant -- see https://mypy.readthedocs.io/en/stable/common_issues.html#variance
bids/ext/reports/parameters.py:80: note: Consider using "Sequence" instead, which is covariant
bids/ext/reports/parsing.py:14: error: Cannot find implementation or library stub for module named "bids.layout"  [import]
bids/ext/reports/parsing.py:107: error: Module has no attribute "func_info"  [attr-defined]
bids/ext/reports/parsing.py:143: error: Module has no attribute "anat_info"  [attr-defined]
bids/ext/reports/parsing.py:174: error: Incompatible types in assignment (expression has type "int", variable has type "str")  [assignment]
bids/ext/reports/parsing.py:185: error: Module has no attribute "dwi_info"  [attr-defined]
bids/ext/reports/parsing.py:228: error: Module has no attribute "fmap_info"  [attr-defined]
bids/ext/reports/parsing.py:253: error: Module has no attribute "meg_info"  [attr-defined]
bids/ext/reports/parsing.py:362: error: Incompatible return value type (got "Optional[FileBasedImage]", expected "Optional[Nifti1Image]")  [return-value]
bids/ext/reports/report.py:14: error: Cannot find implementation or library stub for module named "bids.layout"  [import]
bids/ext/reports/cli.py:15: error: Cannot find implementation or library stub for module named "bids.layout"  [import]
bids/ext/reports/cli.py:15: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 21 errors in 6 files (checked 10 source files)
```

Reply:
Can be re-added later, but fixing these issues feels out-of-scope for this PR.

Reply:
totally outside of the scope indeed